react-frontend/src/components/FaqAccordion/Accordion.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+
+const Accordion = ({ que, ans }) => {
+  const [show, setShow] = useState(false)
+
+  return (
+    <>
+      <div className='main-heading'>
+        <p onClick={() => setShow(!show)}>{show ? '➖' : '➕'}</p>
```

Comment:
The last accordion has irregular styling:
![Screenshot 2024-07-05 232827](https://github.com/openmainframeproject/software-discovery-tool/assets/120312681/c0b9cd07-d95a-4616-90f4-50e6221718c7)

Reply:
For me, it looks fine. It's not responsive yet, so it might appear differently on smaller screens.

Reply:
Just want to add, I viewed it on laptop itself @Apurv428

Reply:
Laptop screen size?

Reply:
14 inches.